frontend/src/components/abfragen/AbfrageNavigationTree.vue

```diff
@@ -1065,6 +1087,21 @@ export default class AbfrageNavigationTree extends Mixins(AbfrageSecurityMixin)
     );
   }
 
+  private isItemTypeOfBaugebietAndBauratenAreDeterminable(abfrageTreeItem: AbfrageTreeItem): boolean {
+    return (
+      this.isItemTypeOfBaugebiet(abfrageTreeItem) &&
+      // Prüfen ob die idealtypischen Bauraten ermittelt werden dürfen.
+      this.isEditableWithAnzeigeContextAbfragevariante(abfrageTreeItem.contextAnzeigeAbfragevariante) &&
+      // Entweder müssen die Geschoßläche Wohnen oder die Wohneinheiten gesetzt sein.
+      (!_.isNil(abfrageTreeItem.baugebiet?.gesamtanzahlWe) ||
+        !_.isNil(abfrageTreeItem.baugebiet?.geschossflaecheWohnen)) &&
+      // Das Baugebiet darf keine Bauraten referenzieren.
+      (_.isNil(abfrageTreeItem.baugebiet?.bauraten) || _.isEmpty(abfrageTreeItem.baugebiet?.bauraten)) &&
```

Comment:
Das `isNil` (`_.isNil(abfrageTreeItem.baugebiet?.bauraten)`) kann vermutlich entfernt werden, da  `isEmpty` ebenfalls auf null und undefined prüft und mit true antwortet.

Die Prüfung der Bauabschnitte in Zeile 1083 in Methode `isItemTypeOfAbfragevarianteAndBauratenAreDeterminable` kann dann ebenfalls angepasst werden.

https://lodash.com/docs/#isEmpty

![image](https://github.com/it-at-m/isi-frontend/assets/56300801/fe70fae2-eaea-4901-b461-74decaccd74a)